common/src/main/java/com/frostwire/search/SearchManager.java

```diff
@@ -415,7 +415,7 @@ public SearchPerformer newPerformer(long token, String keywords) {
         }
     };
 
-    public static final SearchEngine TORRENTDOWNLOADS = new SearchEngine("TorrentDownloads.me", AppSettings.SEARCH_TORRENTDOWNLOADS_ENABLED) {
```

Comment:
hmm, we gotta be careful with this..., this might be the way it was named on update.frostwire.com/android, let's see